src/main/java/org/gridsuite/filter/server/dto/IFilterAttributes.java

```diff
@@ -20,4 +21,6 @@ public interface IFilterAttributes {
     java.util.Date getModificationDate();
```

Comment:
import java.util.Date; unstead inline import java.util.Date getModificationDate();  => not harmful ;)